Jellyfin.Api/Controllers/GenresController.cs

```diff
@@ -66,6 +66,7 @@ public class GenresController : BaseJellyfinApiController
         /// <response code="200">Genres returned.</response>
         /// <returns>An <see cref="OkResult"/> containing the queryresult of genres.</returns>
         [HttpGet]
+        [HttpGet("/MusicGenres", Name = "GetMusicGenres-Deprecated")]
```

Comment:
Don't change the operationId but use the `ObsoleteAttribute` instead. This will add the `deprecated` flag in OpenAPI

Reply:
Can't use `Obsolete` or `ObsoleteAttribute` on a single route. It's either or.

Reply:
I thought it would work when I quickly read the Swashbuckle docs. The name is a problem though as it's used by the generated apiclients and this change causes 2 problems
1) the name is different so existing usages of the function break because the function does not exist anymore
2) the dash `-` is an illegal character and at least the java apiclient will break when it finds it

The first it not a big problem since the migration from non-openapi -> openapi is breaking anyway, the second is annoying though.